src/integration.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { WorkOS } from './workos';
+
+describe('WorkOS', () => {
+  const workos = new WorkOS(process.env.WORKOS_API_KEY);
+
+  describe('userManagement', () => {
```

Comment:
cases

- get an invitations that doesnt exist
- list invitations by an email / org that doesn't exist
- list invitations by an org that _does_ exist
- revoke an invitation that doesn't exist. and/or that has already been revoked
- send invitation to a non-email

perhaps you can use the corresponding controller e2e tests for inspiration